src/transport/raw/tests/TestMessageHeader.cpp

```diff
@@ -304,9 +306,10 @@ void TestPayloadHeaderEncodeDecodeBounds(nlTestSuite * inSuite, void * inContext
     }
 }
 
+#define MAX_FIXED_HEADER_SIZE (8 + 8 + 8)
 struct SpecComplianceTestVector
 {
-    uint8_t encoded[8 + 8 + 8]; // Fixed header + max source id + max dest id
+    uint8_t encoded[MAX_FIXED_HEADER_SIZE]; // Fixed header + max source id + max dest id
```

Comment:
comment should go together with the define if we created a define.
Also please use C++ `constespr` for constants.